lib/matobj2.gd

```diff
@@ -924,15 +923,15 @@ DeclareOperation( "Randomize", [ IsRandomSource, IsMatrixOrMatrixObj and IsMutab
 ##  <Returns>nothing</Returns>
 ##
 ##  <Description>
-##  For two vector objects <A>dst</A> and <A>src</A>,
+##  For two vector objects <A>src</A> and <A>dst</A>,
 ##  such that <A>dst</A> is mutable,
-##  and two lists <A>dcols</A> and <A>scols</A> of positions,
+##  and two lists <A>scols</A> and <A>dcols</A> of positions,
 ##  <Ref Oper="CopySubVector"/> assigns the entries
 ##  <A>src</A><C>{ </C><A>scols</A><C> }</C>
 ##  (see <Ref Oper="ExtractSubVector"/>)
 ##  to the positions <A>dcols</A> in <A>dst</A>,
 ##  but without creating an intermediate object and thus
-##  &ndash;at least in special cases&ndash;
+##  &ndash; at least in special cases &ndash;
```

Comment:
According to what I have read, the rule about inserting spaces around n-dashes or omitting them is that one should use the same behaviour throughout, thus making changes in a single file may lead to infinite loops.
(For textual parentheses, I prefer n-dashes to brackets because there are already lots of other brackets which have other meanings. I think spaces before the opening and after the closing n-dash increase the readability.)

Reply:
Happy to reinstate this.  Use of dashes as parentheses is not something I'm familiar with.

Reply:
This occurs in both 26.9.3 and 26.11.5 so reinstated twice.